template.dd

```diff
@@ -1024,8 +1065,15 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         ------
     )
 
+    $(P The result of deduced type parameters does not depend on the
+        order of function arguments.
```

Comment:
Probably clearer: "Type parameter deduction is not influenced by the order of function arguments."